ZPastel.Test/OrderEndpointTests.cs

```diff
@@ -397,5 +411,34 @@ public async Task CreateOrder_WithNegativeAndZeroQuantityInOrderItem_ResponseSta
             var postResponse = await client.PostAsync("api/orders/create", content);
             postResponse.StatusCode.Should().Be(HttpStatusCode.BadRequest);
         }
+
+        [Fact]
+        public async Task DeleteOrder_WithOrderItems_ShouldDeleteTheOrderAndAllOrderItems()
+        {
+            var order = dataContext
+                .Set<Order>()
+                .Where(o => o.Id == 2)
+                .SingleOrDefault();
+
+            order.Should().NotBeNull();
```

Comment:
Please check also the Order Items are not null and there are any entry on it

Reply:
Added order item validation